common/lib/xmodule/xmodule/tests/test_library_content.py

```diff
@@ -499,6 +499,20 @@ def test_student_view(self):
             patched_methods['send_unit_viewed_event'].assert_called_once_with()
             patched_methods['link_current_user_to_children'].assert_called_once_with()
 
+    def test_student_view_no_reviews(self):
```

Comment:
@macornwell Thanks for renaming and moving this test! Please add a docstring that further explains the purpose of this test.